LICENSE-MIT

```diff
@@ -1,4 +1,6 @@
-Copyright (c) Individual contributors
+The MIT License (MIT)
+
+Copyright (c) 2015-2025 Kevin B. Knapp and Clap Contributors
```

Comment:
Do I revert this change too, going by your comment? If yes, this particular line will be gone from the licenses we symlink. Asking because I wanted to confirm.

Reply:
Yes